data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,585 @@ mission "Heliarchs Buy Jump Drive Mission"
+			choice
+				`	"Why don't you just go back to human space?"`
+					goto stay
+				`	"I'll make sure to do that."`
+			label thanks
+			`	"Thanks again, <first> <last>. I'll forever be grateful to you for helping me. I never thought I'd be successful as an author, but when I met you, you helped me turn my career into something I thought I'd never see."`
+			choice
+				`	"Just helping out."`
+				`	"I'm happy to have helped you."`
+			`	"Again, thank you. You will not be needed anymore for the process."`
+			choice
+				`	(Leave.)`
+			`	You return to the spaceport the way you came and return to your ship. You are slightly anxious about the outcome of the process and whether Anna's book will even make it through the process to be published, but you have more important things to attend to for the next six months.`
+				decline
+			label stay
+			`	"I need to stay here so I can answer their questions during the review process. Don't worry about me."`
+			choice
+				`	"Got it. Good luck publishing your book."`
+					goto thanks
+
+
+event "beyond the hyperdrive is published"
+
+
+mission "Coalition: Struggling Author 5A"
+	landing
+	source
+		not near "Quaru" 0 100
+	destination "Inmost Blue"
+	to offer
+		has "event: beyond the hyperdrive is published"
+		not "Coalition: Struggling Author 5B: offered"
+	on offer
+		conversation
+			`Checking your computer feed for the latest galactic news and trade prices, you get a reminder.`
+			`	You read it, and recall that you had helped an author named Anna Fangvin get her book, 'Beyond the Hyperdrive,' published about half a year ago in Coalition space. You should return to <destination> and check on her.`
+				accept
+	on complete
+		payment 750000
+		conversation "anna invites you to book signing"
+
+mission "Coalition: Struggling Author 5B"
+	landing
+	source
+		near "4 Winter Rising" 1 100
+	destination "Inmost Blue"
+	to offer
+		has "event: beyond the hyperdrive is published"
+		not "Coalition: Struggling Author 5A: offered"
+	on offer
+		conversation
+			`As you land on the spaceport, you check your messages and see a notification from Anna Fangvin, the author who you helped about half a year ago. In the message, she explains that her novel, "Beyond the Hyperdrive," has been approved. You should return to <destination> and meet her.`
+				accept
+	on complete
+		payment 750000
+		conversation "anna invites you to book signing"
+
+
+conversation "anna invites you to book signing"
+	action
+		log "People" "Anna Fangvin" `Anna is an ambidextrous science fiction writer. A struggling author in human space, she found success upon publishing her novel 'Beyond the Hyperdrive' in the Coalition. Her book is a bestseller in the Kimek region.`
```

Comment:
If you say nothing below you don't definitively learn she's ambidextrous. That also feels like a minor note that only comes up in the one line?
```suggestion
		log "People" "Anna Fangvin" `Anna was a struggling science fiction author in human space until she found success publishing her novel 'Beyond the Hyperdrive' in the Coalition. Her book is a bestseller in the Kimek region.`
```